bin/update-ips.bash

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+# This script checks the IPs that are setup for network policies and reports the diff
+# If set to update the config, it will also update the config files.
+
+# Usage: update-ips.bash <cluster> <action>
+#   cluster: What cluster config to check for (sc, wc or both)
+#   action: If the script should update the config or not (update or dry-run)
+
+set -eu -o pipefail
+
+here="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=bin/common.bash
+source "${here}/common.bash"
+
+CHECK_CLUSTER="${1}" # sc, wc or both
+DRY_RUN=true
+if [[ "${2}" == "update" ]]; then
+    DRY_RUN=false
+fi
+has_diff=0
+
+# Compares a list IPs with the list of IPs in the yaml path and file specified and returns the diff return code.
+# If DRY_RUN is set, it will output to stdout, otherwise it will just return the diff return code silently.
+diffIPs() {
+    local yaml_path
+    local file
+    local IPS
+    yaml_path="${1}"
+    file="${2}"
+    shift 2
+    IPS=("$@")
+    tmp_file=$(mktemp --suffix=.yaml)
+
+    yq4 -n '. = []' > "${tmp_file}"
+    for ip in "${IPS[@]}"; do
+        yq4 -i '. |= . + ["'"${ip}"'/32"]' "${tmp_file}"
+    done
+
+    if $DRY_RUN; then
+        out_file=/dev/stdout
+    else
+        out_file=/dev/null
+    fi
+
+    diff \
+        <(yq4 -P "${yaml_path}"' | sort_by(.)' "${file}" || yq4 -n '. = []') \
```

Comment:
```suggestion
        <(yq4 -P "${yaml_path}" // []' | sort_by(.)' "${file}") \
```
This is assumed to be an array right? Then append the lookup with `// []` in case they can be null to set an empty one as default.